default/scripting/buildings/colonies/SP_BANFORO.focs.txt

```diff
@@ -9,22 +9,23 @@ BuildingType
             condition = And [
                 Planet
                 OwnedBy empire = Source.Owner
-                Or [
-                   And [
-                       Species name = "SP_BANFORO"
-                       Population low = [[MIN_RECOLONIZING_SIZE]]
-                       Happiness low = 5
-                    ]
-                    And [
-                        HasSpecial name = "EXTINCT_BANFORO_SPECIAL"
-                        Contains Building name = "BLD_XENORESURRECTION_LAB"
-                    ]
+            Or [
+               And [
+                   Species name = "SP_BANFORO"
+                   Population low = [[MIN_RECOLONIZING_SIZE]]
+                   Happiness low = 5
+                ]
+                And [
+                    HasSpecial name = "EXTINCT_BANFORO_SPECIAL"
+                    Contains Building name = "BLD_XENORESURRECTION_LAB"
                 ]
+            ]
```

Comment:
unintended whitespace changes?

Reply:
Yeah. I didn't manually change this file, I just changed the python script that generates it

Reply:
Checked this. The spacing for the indent of Banforo was edited in master by Geoff over here: https://github.com/freeorion/freeorion/commit/0741af7426f291a1718108d2920725f562d843dc

The spacing for the indent in this commit is consistent with the other extinct species and the spacing generated by col_bld_gen.py

Reply:
Probably col_bld_gen.py should be adjusted to fix the indenting.

My editing the the .focs.txt file was a bad idea and is indicating as such in the comment at the start of the file. So reverting that change is fine.